cookbook/python_sdk.ipynb

```diff
@@ -150,23 +150,21 @@
    "outputs": [],
    "source": [
     "from datetime import datetime\n",
-    "from langfuse.model import InitialGeneration, Usage\n",
-    "\n",
     "generationStartTime = datetime.now()\n",
     "\n",
     "# call to an LLM API\n",
     "\n",
-    "generation = langfuse.generation(InitialGeneration(\n",
+    "generation = langfuse.generation(\n",
     "    name=\"summary-generation\",\n",
-    "    startTime=generationStartTime,\n",
-    "    endTime=datetime.now(),\n",
+    "    start_time=generationStartTime,\n",
```

Comment:
switch to the suggested use of .generation() and .end() to not set dates manually